lib/teleport-rb.rb

```diff
@@ -11,13 +11,46 @@ def initialize auth_server:, identity_file:
     @identity_file = identity_file
   end
 
-  def nodes labels={}
+  private
+
+  def execute_command command, handle_json_errors: true
```

Comment:
Can I get some context on why we *wouldn't* handle JSON errors? (`handle_json_errors`)

Reply:
this is because the configure host function doesn't return json and when you try to handle it, it gives an error. Therefore I provided the `handle_json_errors` flag

Reply:
Cool - then I'd recommend extracting the JSON handling to a function called `execute_json_command` and have that call your `execute_command` function, which means you can get rid of your complicated `handle_json_errors` parameter/logic. 😄 

(And `configure_host` can just call `execute_command` directly.)